assets/css/detours/_detour_map.scss

```diff
@@ -1,5 +1,15 @@
+.c-detour_map--original-route-shape-skeleton {
+  stroke: $color-kiwi-500;
+  opacity: 50%;
```

Comment:
*Note: This doesn't quite match the hi-fi designs, I think. According to Figma, when the route is highlighted, this should be completely opaque, rather than effectively at 85% opacity (or however 50% opacity and 35% opacity stack up. I don't really want to fix that right now though, because keeping it as-is means that it can be fully CSS, while making it fully opaque would involve at least a bit more complexity*